CHANGES.rst

```diff
@@ -92,6 +92,8 @@ ramp_fitting
 - Adding feature to turn off calculations of ramps with good 0th group,
   but all other groups are saturated. [#6737]
 
+- Correcting the default ZEROFRAME allocation. [#6791]
```

Comment:
This entry should go under the datamodels section. Not *everything* is for ramp_fitting! ;-)

Reply:
I was using the six degrees of separation of ramp fitting for the change log, but I'll do it your way 😆